children/models.py

```diff
@@ -0,0 +1,73 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.hashers import make_password
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class Child(models.Model):
+    created_at = models.DateTimeField(verbose_name=_("created at"), auto_now_add=True)
```

Comment:
1. maybe make a `utils.models.TimestampedModel` or something that stores `created_at` and then base other classes off it? since it is used already a couple of times and might be handy elsewhere. 

2. maybe make the `id` -> `uuid` for `Child` might make it easier to update the relations later, but also probably a good thing to hide the number of children in the DB.

Reply:
some `UUIDModel` could also be in `utils.models` if you decide to change to using uuids 🙂

Reply:
1. I'm not sure yet how many and what kind of combos of`created_at` and `updated_at` will be needed in future models. I'll probably add some common base class(es) later when it is more clear what is actually needed :)

2. UUID is indeed way better for that, changing

Reply:
ok, base class(es) can come later, sure 👍